tests/acceptance/test_dcos_package.py

```diff
@@ -24,8 +24,8 @@ def test_uninstall_package_and_wait():
     uninstall_package_and_wait('chronos')
     delete_zk_node('chronos')
     assert package_installed('chronos') == False
-    # this should only pass if we can install after the wait
-    install_package_and_wait('chronos')
+#    # this should only pass if we can install after the wait
+#    install_package_and_wait('chronos')
```

Comment:
@kensipe I can't quite figure out what is causing this to fail now with `dcoscli 0.5.3`... it seems the `wait_for_mesos_task_removal()` isn't properly waiting.